sdk/communication/communication-chat/package.json

```diff
@@ -77,7 +77,7 @@
   },
   "devDependencies": {
     "@azure/core-util": "^1.0.0-beta.1",
-    "@azure/communication-identity": "^1.0.0",
+    "@azure/communication-identity": "^1.1.0-beta.2",
```

Comment:
@petrsvihlik Will Identity be released on 4/5, should I make this version update here?

Reply:
yes